macOS/DuckDuckGo/Localizable.xcstrings

```diff
@@ -34987,7 +34987,7 @@
     },
     "letsmove.alert.message" : {
       "comment" : "Message of the alert shown if the app is launched not from the /Applications folder – suggesting to move it there",
-      "extractionState" : "extracted_with_value",
+      "extractionState" : "manual",
```

Comment:
This was becoming stale for App Store builds because these strings aren't present there.  Setting them to manual prevents them from becoming `stale`.  We could look for an alternative solution though if preferred but this seemed like the simpler approach.